src/utils/smartWallet.js

```diff
@@ -23,10 +23,15 @@ import { TX_CONFIRMED_STATUS } from 'constants/historyConstants';
 
 import type { Accounts } from 'models/Account';
 import type { SmartWalletStatus } from 'models/SmartWalletStatus';
+import type { WalletReducerState } from 'reducers/smartWalletReducer';
```

Comment:
I would rename the type to `SmartWalletReducerState`

Reply:
Updated